cmd/main.go

```diff
@@ -272,6 +273,13 @@ func main() {
 			os.Exit(1)
 		}
 	}
+	// nolint:goconst
+	if os.Getenv("ENABLE_WEBHOOKS") != "false" {
```

Comment:
maybe move ENABLE_WEBHOOKS to a constant?

Reply:
I extracted to a local constant, given it is used only here.